src/assets/css/inc/variables.scss

```diff
@@ -11,16 +11,16 @@ $default-margin: $default-padding;
 $max-content-width: 1366px;
 
 // See: https://mozilla.design/firefox/typography/
-$body-sm-font-size: 14px;
-$body-sm-line-height: 22px;
+$body-sm-font-size: 16px;
+$body-sm-line-height: 1.5;
 $heading-xxs-font-size: 20px;
-$heading-xxs-line-height: 24px;
+$heading-xxs-line-height: 1.2;
 $heading-xs-font-size: 24px;
-$heading-xs-line-height: 28px;
+$heading-xs-line-height: 1.16;
```

Comment:
This is really close to 1.2. I wonder if all of the ones after this could just be 1.2?

As using the unitless line-height provides something that is relative to the font-size, that should mean you can use the same line-height and it will be consistent across font-size changes except for special-cases where you want that ratio to be different like the 1.5 at the start.

Reply:
Yep, let's simplify. I'll keep the different vars in case we want to update some of the line-heights in the future, though (because the guidelines don't have a 1.2 ratio and this is why the values here were all different)